woocommerce/woocommerce-android
# Tracking for "Create Test Order" feature

Part of https://github.com/woocommerce/woocommerce-android/issues/9459

Please do not merge until target base is `trunk`



### Description


This PR adds tracking events for the test order feature:


Event | Triggers
--| --
*_order_list_test_order_displayed | When the empty state with CTA to create test order is displayed.
*_order_list_try_test_order_tapped  | When the Try test order button is tapped on the empty order list screen.
*_test_order_start_tapped  | When the Start test order button is tapped on the create test order screen.


### Testing instructions



1. Start with a test site with no order, at least one payment method, and make sure the site is accessible publicly already,
2. Go to Orders list in the app. Check that the app shows the "Try a test order" empty order screen. Check logcat and see that `order_list_test_order_displayed` is tracked.
3. Tap "Try a test order", check logcat and see that `order_list_try_test_order_tapped` is tracked and the dialog is shown.
5. Tap the "Start test order" button and ensure it opens to the site on a browser window. Check logcat and see that `test_order_start_tapped` is tracked.
